composer.json

```diff
@@ -11,34 +11,32 @@
             "email": "schmittjoh@gmail.com"
         }
     ],
-    "minimum-stability": "dev",
     "require": {
-        "php": "~5.3",
+        "php": "~5.3|~7.0",
         "symfony/framework-bundle": "~2.1",
-        "symfony/security-bundle": "*",
+        "symfony/security-bundle": "~2.0|~3.0",
         "jms/metadata": "~1.0",
         "jms/parser-lib": "~1.0",
         "jms/aop-bundle": "~1.0",
         "jms/di-extra-bundle": "~1.3"
     },
     "require-dev": {
-        "sensio/framework-extra-bundle": "*",
-        "symfony/class-loader": "*",
-        "symfony/yaml": "*",
-        "symfony/browser-kit": "*",
-        "symfony/finder": "*",
-        "symfony/css-selector": "*",
-        "symfony/process": "*",
-        "doctrine/doctrine-bundle": "*",
-        "symfony/twig-bundle": "*",
-        "doctrine/orm": "*",
-        "symfony/form": "*",
-        "symfony/validator": "*"
+        "sensio/framework-extra-bundle": "~2.0|~3.0",
```

Comment:
My comment was confusing... I just meant dropping support for symfony 2.1 and 2.2